src/views/FindPark.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <section>
+    <input
+      class="search-input"
+      type="text"
+      placeholder="Search by name, city or zip code"
+      v-model="searchTerm"
+    />
+    <button class="search-button" @click="search('searchTerm')">
+      Get Started - woof!
+    </button>
+    <p v-if="searchTerm === ''">Please enter a park name, city, or zip code</p>
+  </section>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      searchTerm: '',
+    };
+  },
+  methods: {
+    search() {
+      if (this.searchTerm !== '') {
+        console.log(this.searchTerm);
```

Comment:
Let's get this `console.log` out of here!

Reply:
Deleted! I left it in just so we could see that the button was connected and working, but we can always add it back if needed.

Reply:
Thanks, Stacy! That makes sense.